frontend/src/css/modules/leaderboard.scss

```diff
@@ -3,3 +3,11 @@
 .horizontal-scroll {
 	overflow-x: auto;
 }
+
+
+.baseline-badge {
+	font-size: 1rem !important;
+	font-weight: normal !important;
+	float:right !important;
+	margin-right: 40% !important;
+}
```

Comment:
Please add a new line here. Also, using `!important` isn't a good practice. Can you please remove it and do it the right way.